tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -121,6 +123,7 @@ def test_set_experiment_tag_of_non_active_experiment(init_store):
         assert "must be in the 'active' state" in str(excinfo.value)
```

Comment:
Isn't this test and experiment feature ?

Reply:
set_experiment_tag function is not implemented yet